CMakeLists.txt

```
@@ -431,7 +433,10 @@ endif()
 add_subdirectory(python)
 
 # Add 'ALL' target that builds everything
-set(all_target_candidates tests_and_run dartpy pytest)
+set(all_target_candidates dartpy)
+if (BUILD_TESTING)
+  set(all_target_candidates tests_and_run pytest)
```

Maybe add instead of replace?

```suggestion
  list(APPEND all_target_candidates tests_and_run pytest)
```

---

oops, thanks! I applied the suggestion